src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -22,11 +22,23 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.File.ExportToCSV</oldName>
-    <newName>DSCore.IO.File.ExportToCSV</newName>
+    <newName>DSOffice.Data.ExportCSV</newName>
```

Comment:
Do we need to add another migration entry for `DSCore.IO.File.ExportToCSV` to `DSOffice.Data.ExportCSV`?